### PR TITLE
restore anniversaries.json

### PR DESCRIPTION
**Pull request by pinkywafer**
requirements version added in 4.2.2

### CODE REVIEW

**Comment by ludeeus** [comment]
You need to rebase your branch (and you should really not be using your `master` branch when making PR's :smiley:  )

File: components/anniversaries.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{	
+  "name": "anniversaries",	
+  "owner": ["pinkywafer"],
```
Comment (ludeeus) [inline components/anniversaries.json:3]:
```suggestion
  "owner": ["@pinkywafer"],	
```

Comment (pinkywafer) [reply components/anniversaries.json:3]: I got it! - just a sec!!

**Review by ludeeus — APPROVED** [review]
Thanks @pinkywafer :+1:

**Pull request merged**